data/fields/historic.json

```diff
@@ -4,28 +4,45 @@
     "label": "Type",
     "strings": {
         "options": {
+            "aircraft": "Aircraft",
             "archaeological_site": "Archaeological Site",
+            "battlefield": "Historic Battlefield",
             "bomb_crater": "Bomb Crater",
             "boundary_stone": "Boundary Stone",
+            "bridge": "Historic Bridge",
             "building": "Historic Building",
+            "bunker": "Historic Bunker",
+            "cannon": "Historic Cannon",
             "castle": "Castle",
+            "cemetery": "Cemetery",
```

Comment:
I see that that `historic=bridge` got "Historic Bridge" label and `historic=cemetery` got "Cemetery" label.

What is the logic behind this?